docs/latest/tutorial/code-signing.md

```diff
@@ -1,37 +1,49 @@
 ---
-title: "Code Signing"
-description: "Code signing is a security technology that you use to certify that an app was created by you."
+title: 'Code Signing'
+description: 'Code signing is a security technology that you use to certify that an app was created by you.'
 slug: code-signing
 hide_title: false
 ---
 
-# Code Signing
+:::info Tutorial parts
```

Comment:
IMO the full code-signing doc should be separate from the tutorial. I feel like this is too long and has too many options to be part of the step-by-step tutorial.

I'm suggesting two alternative presentations for this info:
* A subsection of "Application Distribution" that references this doc and explains the basics of code-signing.
* Replacing this doc with a step-by-step tutorial on Windows and macOS code-signing in Electron Forge, and leaving the previous code-signing doc alone.